### ISSUE TITLE
Regarding the weights initialization

### ISSUE THREAD

**Issue by WenChouWu** [2019-dl-training-program/Lab2#5]
Hi TAs,

I try to add codes for weights initialization but observes that both the training and validation error raises with below codes. I can not figure out the problem after doing some searchings in Google. Could you help? Thank you.

    .....
    model=net.to(device)
    model.apply(weights_init)
    .....

    def weights_init(m):
        classname = m.__class__.__name__
        if classname.find('Conv') != -1 or classname.find('Linear') != -1:
            nn.init.kaiming_normal_(m.weight, mode='fan_out', nonlinearity='relu')
        elif classname.find('BatchNorm') != -1:
            nn.init.constant_(m.weight, 1)
            nn.init.constant_(m.bias, 0)

**Comment by ZmaraXion** [comment]
Hello, 
You can refer to the official implementation(pytorch):
https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py#L154

The conv layer in pytorch has already used the kaiming_normal initialization method when it is declared. So, actually, you don't need to do any additional initialization.
reference: [pytorch source code](https://pytorch.org/docs/stable/_modules/torch/nn/modules/conv.html)
